test/commands/environments/delete.test.js

```diff
@@ -6,37 +6,51 @@ const fancy = test.register('nock', Nock);
 const EnvironmentSerializer = require('../../../src/serializers/environment');
 
 describe('environments:delete', () => {
-  fancy
-    .stdout()
-    .env({ SERVER_HOST: 'http://localhost:3001' })
-    .nock('http://localhost:3001', api => api
-      .get('/api/environments/324')
-      .reply(200, EnvironmentSerializer.serialize({
-        id: '324',
-        name: 'Staging',
-        apiEndpoint: 'https://forestadmin-server-staging.herokuapp.com',
-        active: true,
-        type: 'development',
-        liana: 'forest-express-sequelize',
-        version: '1.3.2',
-        FOREST_ENV_SECRET: '2c38a1c6bb28e7bea1c943fac1c1c95db5dc1b7bc73bd649a0b113713ee29125',
-      })))
-    .nock('http://localhost:3001', api => api
-      .delete('/api/environments/324')
-      .reply(204))
-    .command(['environments:delete', '324', '-p', '82', '--force'])
-    .it('should delete the environment', (ctx) => {
-      expect(ctx.stdout).to.contain('Environment Staging successfully deleted.');
-    });
+  describe('on an existing environment', () => {
+    fancy
+      .stdout()
+      .env({ SERVER_HOST: 'http://localhost:3001' })
+      .nock('http://localhost:3001', api => api
+        .get('/api/environments/324')
+        .reply(200, EnvironmentSerializer.serialize({
+          id: '324',
+          name: 'Staging',
+          apiEndpoint: 'https://forestadmin-server-staging.herokuapp.com',
+          active: true,
+          type: 'development',
+          liana: 'forest-express-sequelize',
+          version: '1.3.2',
+          FOREST_ENV_SECRET: '2c38a1c6bb28e7bea1c943fac1c1c95db5dc1b7bc73bd649a0b113713ee29125',
+        })))
+      .nock('http://localhost:3001', api => api
+        .delete('/api/environments/324')
+        .reply(204, {
+          meta: {
+            job_id: 78,
+          },
+        }))
+      .nock('http://localhost:3001', api => api
+        .get('/api/jobs/78')
+        .reply(200, {
+          state: 'complete',
+          progress: '100',
+        }))
+      .command(['environments:delete', '324', '-p', '82', '--force'])
+      .it('should delete the environment', (ctx) => {
+        expect(ctx.stdout).to.contain('Environment Staging successfully deleted.');
+      });
+  });
```

Comment:
Could we have a test with an active state and then se the server to reply complete or failed to see if it works correctly ?